Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -63,8 +63,8 @@ DrawMol::DrawMol(
       yMax_(std::numeric_limits<double>::lowest() / 2.0),
       xRange_(std::numeric_limits<double>::max()),
       yRange_(std::numeric_limits<double>::max()),
-      flexiCanvasX_(width_ < 0.0),
```

Comment:
I won't deny that it's clearer to the casual user that flexiCanvasX_ is being initialised correctly here, but the original code is correct, as explained at https://en.cppreference.com/w/cpp/language/constructor. Variables are always initialised in the order they are declared in the class.  If someone changes the order in the class such that width_ is after flexiCanvasX_, the compiler should spit out an error at this point to the effect that width_ is being used before it is initialised.

Reply:
agreed. My concerns about that were incorrect. I do this is somehow more readable in that it makes the direct connection between the argument and the value of `flexiCanvasX_`